addons/website_forum/views/website_profile_templates.xml

```diff
@@ -100,9 +111,18 @@
                                 <t t-call="website_forum.display_post_answer"/>
                             </div>
                         </t>
-                        <t t-else="">
-                            <span class="fw-bold">No answer posted yet.</span>
-                        </t>
+                        <div t-elif="my_profile" class="text-muted">
+                            You have not answered any questions yet. <br />
```

Comment:
Too much "s"

Reply:
In this case we're referring to multiple questions, this "s" can stay.